src/phoenix/trace/span_evaluations.py

```diff
@@ -152,6 +157,24 @@ def __init_subclass__(
             tuple(sorted(prod)) for prod in product(*cls.index_names.keys())
         )
 
+    def to_parquet(self, path: Union[str, Path]) -> None:
```

Comment:
path should be optional

Reply:
If the caller does not know the path, how would they restore the evaluation if there are multiple present?

Reply:
I was thinking it could have an alternative signature:

```python
to_parquet(self, path: optional, filename: optional) -> full_path:
```
so that the caller doesn't have to fill this out. The return value would give them a way to restore.